data/source/MongoDb.php

```diff
@@ -30,21 +40,39 @@
  * ```
  * // config/bootstrap/connections.php:
  * Connections::add('default', ['type' => 'MongoDb', 'database' => 'myDb']);
+ * Connections::add('default', [
+ *     'type'     => 'MongoDb',
+ *     'database' => 'myDb',
+ *     'host'     => 'example.org:27017',
+ *     //'host'   => [host1:27017, host2:27018], //multiple hosts
+ *     'uriOptions' => ['replicaSet' => 'Cluster0'],
+ * ]);
+ * ```
+ *
+ * or using a DSN:
+ * ```
+ * // config/bootstrap/connections.php:
+ * Connections::add('default', [
+ *     'type'     => 'MongoDb',
+ *     'database' => 'test',
+ *     'dsn'      => 'mongodb://c0-s1.mongodb.net:27017,c0-s2.mongodb.net:27017',
+ *     //'dsn'      => 'mongodb+srv://cluster0-test.mongodb.net',
+ * ]);
```

Comment:
First, I suggest using the same example values as the default ones we provide in https://github.com/UnionOfRAD/framework/blob/master/app/config/bootstrap/connections.php#L44
`myDb` -> `my_app`.

We shouldn't mention file locations `config/bootstrap/connections.php` in the API docs of the framework's core classes, I know one mention was in there before. Distros/apps  do not necessarily keep their configurations at the same place. These locations are standardized inside the distros only.

To show variations of configurations, I suggest to first show a full example like you do, than just pick out that single line that changes. This makes the docblock shorter and quicker to read.

```
Connections::add('default', [
    'type'     => 'MongoDb',
    'database' => 'myDb',
     'host'     => 'example.org:27017',
     'uriOptions' => ['replicaSet' => 'Cluster0'],
]);
```

Or with multiple hosts:
```
'host'   => [host1:27017, host2:27018],
```

Third, can you split up the first basic example into two fenced code blocks? That makes it clear that these are distinct cases.